### ISSUE TITLE
Observatory doesn't show any asteroids when scanning

### ISSUE THREAD

**Issue by azmicomico** [zmaster587/AdvancedRocketry#928]
Minecraft Version:
1.10.12
Forge Version:
1.10.2.12.18.3.2511
LibVulpes Version:
1.10.12-0.2.6-28
AdvancedRocketry Version:
1.10.12-1.2.5-51

When you click 'Scan' on the observatory, no asteroids show up on the list. Tried reverting to default asteroidConfig.xml, but the behaviour is the same. This was working with AdvancedRocketry-1.10.12-1.2.5-48.

**Comment by Lyneira** [comment]
I have this bug in 1.12 as well. The observatory takes 100 distance data but does not produce an asteroid list. This unfortunately also makes it impossible to test whether the earlier fix for mining missions works because it's impossible to produce asteroid ID chips.

Minecraft Version:
1.12.2
Forge Version:
1.12.2-14.23.1.2555 (recommended build as of this writing)
LibVulpes Version:
1.12.2-0.2.6-16
AdvancedRocketry Version:
1.12.2-1.2.6-25

I have tried this in both a custom modpack as well as a test instance with only AR and LibVulpes installed, both with a custom asteroidConfig.xml and after deleting it with the default generated one. I have also repeatedly tried breaking the observatory block and replacing it to see if that would fix the asteroid list, but it did not. (it used to when I was changing asteroid configs in a previous version)

In case it matters any, I was testing this on a space station oriented away from the sun but in my modpack world where I ran into the problem the observatory is on earth.
